railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

```diff
@@ -35,10 +35,6 @@ Rails.application.configure do
 
   <%- end -%>
   <%- unless options.skip_action_mailer? -%>
-  # Disable caching for Action Mailer templates even if Action Controller
-  # caching is enabled.
-  config.action_mailer.perform_caching = false
```

Comment:
I think we want to keep this false, no? Line 23 make action_controller false.

Reply:
I thought we only needed the action mailer config if it differed from action controller? But if not, then let's add it back up in the same block.

Reply:
Turns out we actually are fine with caching in test. Feature not bug.